service-loadbalancer/service_loadbalancer.go

```diff
@@ -430,6 +430,12 @@ func main() {
 
 	var kubeClient *client.Client
 	var err error
+
+	_, err = newSyslogServer("/var/run/haproxy.log.socket")
```

Comment:
Can you please add a -v flag that can turn this off in case it causes disk problems?